### PR TITLE
Define all Cookie arguments explicitly

### PR DESCRIPTION
**Pull request by iambrosi**
This adds support for Symfony 4.2, which deprecates not defining all the arguments as some of their default values are scheduled to change in Symfony 5.0.

### CODE REVIEW

**Comment by iambrosi** [comment]
Any news on this?

**Comment by iambrosi** [comment]
@dunglas Could you please take a look at this?

File: EventListener/AngularCsrfCookieListener.php [hunk 1]
```diff
@@ -102,7 +102,9 @@ public function onKernelResponse(FilterResponseEvent $event)
             $this->cookiePath,
             $this->cookieDomain,
             $this->cookieSecure,
-            false
+            false /* httpOnly */,
+            false /* raw */,
+            null  /* sameSite */
```
Comment (dunglas) [inline EventListener/AngularCsrfCookieListener.php]: Shouldn't we set this flag to `lax` at least?

Comment (iambrosi) [reply EventListener/AngularCsrfCookieListener.php]: Yes, I agree, security first. I set it to `null` initially as this was the default behavior in previous versions. This has been done.

Comment (iambrosi) [reply EventListener/AngularCsrfCookieListener.php]: @dunglas This value is now set to `lax` instead of `null`.

**Comment by iambrosi** [comment]
@dunglas Is this ready to be merged?

**Comment by dunglas** [comment]
Can you check why Travis is red?

**Comment by iambrosi** [comment]
Travis is not red. AppVeyor is. It cannot find the path to `c:\tools\php72`.

**Comment by dunglas** [comment]
Can you try to replace php72 by php73 please? It should fix the build.

**Comment by sparksterz** [comment]
@iambrosi you can probably revert that last commit and just change the `cinst -y php` line to `cinst -y php --version 7.2.17`

Looks like even in your earlier builds it was pulling 7.3 and then failing, and when upgrading a dependency is looking for PHP < 7.3

**Comment by iambrosi** [comment]
@dunglas Tests in AppVeyor have been fixed. Thanks @sparksterz for the tip! Also, fixed a new spec failure by a new method call added in https://github.com/symfony/symfony/pull/29944

**Comment by matyax** [comment]
Bump @dunglas. Is there any blocker to merging this PR?

**Comment by dunglas** [comment]
Thanks @iambrosi

**Pull request merged**